cmake/mcuboot.cmake

```diff
@@ -120,7 +120,7 @@ if(CONFIG_BOOTLOADER_MCUBOOT)
       ${PYTHON_EXECUTABLE}
       ${NRF_DIR}/scripts/bootloader/zb_add_ota_header.py
       --application ${PROJECT_BINARY_DIR}/app_update.bin
-      --application-version-string ${CONFIG_ZIGBEE_OTA_APP_VERSION}
```

Comment:
Is the `CONFIG_ZIGBEE_OTA_APP_VERSION` still needed?

Reply:
No, it is not. It is fully replaced with the `MCUBOOT_IMAGE_VERSION`.